### PR TITLE
[`from_single_file()`]fix: local single file loading.

### PR DESCRIPTION
**Pull request by sayakpaul**
Fixes https://github.com/huggingface/diffusers/issues/5390.

In favor of closing: https://github.com/huggingface/diffusers/pull/5439. Copying over what I said in #5439: 

TL;DR: It's a must for the user to locally save the tokenizers and configurations under the same paths (such as "openai/clip-vit-large-patch14") to ensure local_files_only=True works in the right manner.

Here is an end-to-end demonstration of how this should be done:
https://colab.research.google.com/gist/sayakpaul/b086a1089f6723d3aa01b67f77f1ebbc/scratchpad.ipynb

### CODE REVIEW

File: src/diffusers/loaders.py [hunk 1]
```diff
@@ -2832,6 +2832,7 @@ def from_single_file(cls, pretrained_model_link_or_path, **kwargs):
             tokenizer=tokenizer,
             original_config_file=original_config_file,
             config_files=config_files,
+            local_files_only=local_files_only,
```
Comment (sayakpaul) [inline src/diffusers/loaders.py:2835]: We never actually passed it to the method.

**Review by patrickvonplaten — APPROVED** [review]
Very nice! Thanks for the fix

**Comment by Louis24** [comment]
> Fixes #5390.
> 
> In favor of closing: #5439. Copying over what I said in #5439:
> 
> TL;DR: It's a must for the user to locally save the tokenizers and configurations under the same paths (such as "openai/clip-vit-large-patch14") to ensure local_files_only=True works in the right manner.
> 
> Here is an end-to-end demonstration of how this should be done: https://colab.research.google.com/gist/sayakpaul/b086a1089f6723d3aa01b67f77f1ebbc/scratchpad.ipynb

yep, thank you for this info,could specify the path like: "C:\Zero\Python\AIGC\Stable Diffusion WebUI Q v4.4\.cache\huggingface\hub\models--openai--clip-vit-large-patch14"?

**Comment by olegchomp** [comment]
Trying to build local pipeline and can't get idea why we can't change path to clip? For ex. if i have structured project and want folder with config.json be somewhere apart of root project path?
